akka-sample-sharding-java/src/main/resources/application.conf

```diff
@@ -0,0 +1,22 @@
+akka {
+  actor {
+    provider = "cluster"
+  }
+  remote {
+    log-remote-lifecycle-events = off
+    netty.tcp {
+      hostname = "127.0.0.1"
+      port = 0
+    }
+  }
+
+  cluster {
+    seed-nodes = [
+      "akka.tcp://ShardingSystem@127.0.0.1:2551",
+      "akka.tcp://ShardingSystem@127.0.0.1:2552"]
+
+    # auto downing is NOT safe for production deployments.
+    # you may want to use it during development, read more about it in the docs.
+    auto-down-unreachable-after = 10s
+  }
+}
```

Comment:
It'd be really neat if we could have sbt take care of this instead of relying on our editors.